rom/laghos_rom.hpp

```diff
@@ -169,6 +169,7 @@ struct ROM_Options
     bool SNS = false; // if true, use SNS relation to obtain nonlinear RHS bases by multiplying mass matrix to a solution matrix. See arXiv 1809.04064.
     double energyFraction = 0.9999; // used for recommending basis sizes, depending on singular values
     double energyFraction_X = 0.9999; // used for recommending basis sizes, depending on singular values
+    bool shift_sv = false; // if true, shift energy fraction calculation when window-dependent offsets are not used
```

Comment:
Should this be an `int` type, so that any shift could be specified (default 1)? Also, it would be helpful to have a more detailed comment, perhaps here, explaining why the shift is needed.